docs/streams/kafka_on_gdn/fluent-bit-client.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 60
+title: Fluent Bit Client
+---
+
+Fluent Bit is an open-source and multi-platform log processor tool that aims to be a generic Swiss knife for logs processing and distribution.
+
+Fluent Bit works as a logging pipeline chunked into logical parts like Inputs, Parsers, Filters and Outputs. Kafka client implementation able in Output part. It means Fluent bit produces records and sends to Kafka.
```

Comment:
Review capitalization in this sentence. Also, use the serial comma (comma before the and).